### DISCUSSION TITLE
React Native support?

### DISCUSSION THREAD

**Discussion by hyochan** [alephjs/aleph.js#310] [Q&A] [answered]
Will there be any future updates for react native?

**Comment by hazae41** [comment]
There can be a third party module

**Comment by shadowtime2000** [comment, marked as answer]
There is an already open feature request for react native support, #205. It hasn't been discussed much but if react native support does get added it will most likely be very in the future much after aleph goes stable.

**Reply by hyochan** [reply]
Sounds great. Looking forward to seeing the stable version of `aleph`!
